examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -0,0 +1,853 @@
+{
```

Comment:
<p>Now it's time (drop comma)</p><p> </p><p>"the values of the function ..." perhaps <code>Function</code> ? or maybe just <code>f</code> ? since that's its name in the running example.</p><p> </p><p>also, <code>cy</code>  or <code>ci</code>  ? below you use <code>ci</code>  ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>